### ISSUE TITLE
Command "GoTo" raised error

### ISSUE THREAD

**Issue by CameronBanga** [fman-community/fman#373]
Getting this error when I open most folders on my machine:

![screen shot 2018-04-23 at 3 25 54 pm](https://user-images.githubusercontent.com/611354/39151390-b7c1483a-470a-11e8-88a4-f99b7bcc46a4.png)

```
Command 'GoTo' raised error.

Traceback (most recent call last):
  File "fman/impl/plugins/plugin.py", line 289, in _run_in_thread
  File "/Applications/fman.app/Contents/Resources/Plugins/Core/core/commands.py", line 874, in __call__
    self.pane.set_path(url)
  File "fman/__init__.py", line 116, in set_path
  File "fman/impl/widgets.py", line 106, in set_location
  File "fman/impl/model/__init__.py", line 42, in set_location
ValueError: 'Size' is not in list
```

Here's my version info:

```
fman Version:	1.0.1
fman Api Version:	unknown
Python Version:	3.5.4
```

If I can provide any other info let me know!

**Comment by mherrmann** [comment]
Sorry for the trouble Cameron. Does the problem go away when you delete the file `/Users/<username>/Library/Application Support/fman/Plugins/User/Settings/Sort Settings (Mac).json`?

**Comment by CameronBanga** [comment]
@mherrmann That seems to have fixed it! I couldn't tell from error message as to what Plugin it was, and didn't want to delete one by one till I found it. 

Just for my own knowledge, was I missing something obvious I the error report above, or was this something you just need to know from working on the app?

**Comment by mherrmann** [comment]
Happy to hear that it works now. I definitely knew where to look because I know fman so well. Were you on an old version of fman before yesterday?

**Comment by CameronBanga** [comment]
Probably, I'm not sure, as I didn't check until after I saw the issue. 

Been out of town and haven't used fman in probably 2-3 weeks? I'm not sure what the old version was TBH.

**Comment by mherrmann** [comment]
Okay, no problem. I'm glad it's fixed. I'll see what I can do to make fman more resilient with respect to this. Thanks!

**Comment by mherrmann** [comment]
I just released [fman 1.0.2](https://fman.io/changelog#v1.0.2) that gracefully handles this problem :-)
